open-qr-standard.md

```diff
@@ -33,6 +33,8 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 |Code Provider|The system that represents the initiating side of the user journey.  This is the system that creates and publishes the QR Code.  For a point-of-sale payment this would be the merchant checkout system.|
 |Code Consumer|The system that represents the acceptance, or client, side of the user journey.  This is the system that scans and processes the QR Code.  For a point-of-sale payment this would be the customer’s mobile wallet application.|
 |QR Profile|A profile for defining the format of the various actions and meta data that can be transferred using this standard for a specific type of use case or user journey.|
+|Offer|The information about the profiles and capabilities associated with a specific QR Code.|
+|Session|An established connection between a QR Provider and a QR Consumer.|
```

Comment:
Suggestion: "Details about an offer that has been claimed by a QR Consumer".

Reply:
Suggestion: "An established connection (a claim on an offer) between a QR Provider and QR Consumer"